bin/k3s-modsync.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
```

Comment:
Can you make this 4 space indentation as well as rename the file to `k3s_modsync.sh`